src/main/java/com/minecraftcorp/lift/bukkit/listener/PlayerListener.java

```diff
@@ -164,6 +164,35 @@ public void onPlayerMove(PlayerMoveEvent event) {
 		player.setVelocity(pushBack);
 	}
 
+	@EventHandler
+	public void onPlayerChangedWorldEvent(PlayerChangedWorldEvent event) {
+		Player player = event.getPlayer();
+		Location playerLocation = player.getLocation();
+
+		boolean playerInNoLift = plugin.isInNoLift(player.getUniqueId());
+
+		if (playerInNoLift) {
+			return;
+		}
+
+		BukkitElevator elevator = plugin.getActiveLifts()
+				.stream()
+				.findFirst()
+				.orElse(null);
+
+		if (elevator == null) {
+			return;
+		}
```

Comment:
We should keep in mind that there could be more than one current active lifts on the server. 
So we need to check for each active lift if the player is part of that one in order to find the correct instance for removing the passengers/freezers later.
I could imagine to have sth like this:
```
plugin.getActiveLifts()
	.stream()
	.filter(elevator -> elevator.getPassengers().contains(player) || elevator.getFreezers().contains(player))
	.forEach(elevator -> {
		elevator.removePassengers(Collections.singletonList(player));
		elevator.removeFreezers(Collections.singletonList(player));
	});
```

Reply:
I think my implementation is stuck when multiple lifts are active.
Thank you kindly for the example implementation, and fix it.